server/routes/profile.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const multer = require("multer");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { savePhoto } = require("../controllers/profile");
+
+const upload = multer({dest: './uploads'});
+
+// router.route("/photo").post(protect, savePhoto);
+router.route("/savePhoto").post(upload.fields([{name: 'photos'}]), savePhoto);
```

Comment:
Good implementation. You may want to break `.post(...` to the line to make for better readability.